plugins/kotlin/j2k/shared/tests/testData/newJ2k/typeParameters/classDoubleParametrizationWithTwoBoundsWithExtending.kt

```diff
@@ -1,4 +1,4 @@
 internal interface INode
 internal interface Node
 internal open class A
-internal class CC<T, K> : A() where T : INode?, T : Comparable<T?>?, K : Node?, K : Collection<K?>?
```

Comment:
IMO the nullability change on `T` is good--I think removing the out/in earlier allows some of the nullability postprocessing steps to function at their best, since it's still `T?` before any postprocessing steps run (see attached screenshot). That being said, maybe I'm missing some important cases where it should stay nullable?
![Screenshot 2024-04-21 at 9 16 29 PM](https://github.com/JetBrains/intellij-community/assets/53094167/aa374f6c-e1d0-45fd-a028-41f139529445)